src/wallet/rpcwallet.cpp

```diff
@@ -3066,6 +3069,8 @@ UniValue listunspent(const JSONRPCRequest& request)
     CAmount nMaximumAmount = MAX_MONEY;
     CAmount nMinimumSumAmount = MAX_MONEY;
     uint64_t nMaximumCount = 0;
+    CCoinControl coinControl;
```

Comment:
nit: most of the code base use coin_control as variable name, i think it's good use the same pattern. Not sure if there is a variable name standard for the codebase

Reply:
Hmm, it doesn't look like the code-base has a standard, looking at the whole codebase, both are used quite actively, and in rpcwallet.cpp both are used.